### PR TITLE
[DNM]  WIP : new round of jewel mailmap updates

### PR DESCRIPTION
**Pull request by dupont-y**
Do Not merge, work in progress.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -7,8 +7,8 @@
 #
```
Comment (theanalyst) [inline .mailmap]: commit message missing a singed-off-by, otherwise looks good

**Comment by dupont-y** [comment]
Corrected the forgotten signed-off & added a jenkins modification. This is the base work for the official mail for v10.2.0

**Comment by theanalyst** [comment]
Sent mails to authors ?

**Comment by dupont-y** [comment]
Le 25/04/16 10:45, Abhishek L a écrit :
>
> Sent mails to authors ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/ceph/ceph/pull/8659#issuecomment-214211672>
>
Hi Abhishek ,

Pushed all this a little in hurry I confess, then stayed disconnected 
all this week-end.
Will do this evening. Luckily the patch set is very small.
Sorry for the delay.

Cheers,

**Comment by theanalyst** [comment]
@dupont-y no problems, yes a small changeset this time (only 2-3 authors in unaffiliated) :)

**Comment by dupont-y** [comment]
Mails have been sent. Once again, sorry for the delay

**Comment by dupont-y** [comment]
Mails were sent. Got no answers back, I suppose it's ok to merge :)

**Comment by theanalyst** [comment]
@dupont-y now we need a rebase :)

**Comment by dupont-y** [comment]
:(
I See.
Rebasing is difficult because the 1st patch was a sort.

I'll probably end up doing another pull request.

**Comment by dupont-y** [comment]
#9182 should be used instead of this one.

**Pull request closed without merging**